source/mainnet/net/references/query-node.rst

```diff
@@ -318,3 +319,21 @@ following command:
 
 This prints a JSON list containing any such pending updates. As before you can
 also pass a block hash to the command to query a specific block.
+
+Baker queries
+==============
+
+Earliest time a baker may be expected to bake a block
+-----------------------------------------------------
+
+.. code-block:: console
+
+    $concordium-client baker win-time 1
+    Baker 1 is expected to bake no sooner than:
+    Thu, 26 Oct 2023 07:01:26 UTC  (in 34s 699ms)
+
+Show the earliest time the baker may be expected to bake a block.
```

Comment:
Can you put some more words to this. There are quite a lot of qualifiers needed here about the meaning of the response. We have some of this in the API documentation.